rpcs3/Emu/FS/VFS.cpp

```diff
@@ -379,7 +381,35 @@ vfsDevice* VFS::GetDevice(const std::string& ps3_path, std::string& path) const
 		}
 
 		if (max_i < 0)
-			return nullptr;
+		{
```

Comment:
See https://github.com/tambry/rpcs3/commit/7869fd6c5a727a867667cd14a8ef76f45e8a48f5 commit message for the explanation.